cw_decred/lib/wallet_service.dart

```diff
@@ -94,10 +80,12 @@ class DecredWalletService extends WalletService<
     final wallet = await DecredWalletBase.create(
         password: credentials.password!,
         mnemonic: credentials.mnemonic,
-        walletInfo: credentials.walletInfo!,
-        unspentCoinsInfo: unspentCoinsInfoSource);
-    await wallet.save();
-    await wallet.init();
+        walletInfo: credentials.walletInfo!);
     return wallet;
   }
+
+  @override
+  Future<DecredWallet> restoreFromKeys(
+          DecredRestoreWalletFromWIFCredentials credentials) async =>
+      throw UnimplementedError();
```

Comment:
It appears this may be enough, based on my previous comment. The assert there may not be necessary. Is it even possible to not have that `DecredRestoreWalletFromWIFCredentials` class at all and just use `WalletCredentials` directly instead?

Reply:
The compiler was complaining without putting something there in the constructor I think. Also maybe can use for watching only if we have that? Leaving for now with the throws.